developer-docs-site/docs/aptos-developer-domains.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Aptos Developer Domains"
+slug: "aptos-developer-domains"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Aptos Developer Domains
```

Comment:
Let's make this the top level index of getting started? and then have links at the top that drive you toward other areas?

I also worry we might be spending too much time thinking through this and not enough on the overall story. We can easily bikeshed... I choose yellow.